src/scss/main.scss

```diff
@@ -38,3 +38,46 @@ pre {
 .page-title {
   @apply text-3xl md:text-5xl font-bold leading-9 mb-6;
 }
+
+.va-modal {
```

Comment:
Would be nice to have before/after side by side, so that we can show the problem to designer and figure out what we have to update in ui, and where design can be set aside.

Something like this:

ui
[input]
admin
[input]